config/checkstyle-input-suppressions.xml

```diff
@@ -1617,96 +1617,6 @@
             files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck3.java"/>
   <suppress id="UnnecessaryOkComment"
             files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck3.java"/>
-  <suppress id="UnnecessaryOkComment"
```

Comment:
>proof that all suppressions are removed :

Please explain why `grep` shows one result and has an exit code of zero if all suppressions are removed.

Reply:
We should be removing ok comments from all input files associated with the given check.